app/vmagent/remotewrite/client_test.go

```diff
@@ -0,0 +1,94 @@
+package remotewrite
+
+import (
+	"math"
+	"net/http"
+	"testing"
+	"time"
+)
+
+func TestCalculateRetryDuration(t *testing.T) {
+	// `testFunc` call `calculateRetryDuration` for `n` times
+	// and evaluate if the result of `calculateRetryDuration` is
+	// 1. >= expectMinDuration
+	// 2. <= expectMinDuration + 10% (see timeutil.AddJitterToDuration)
+	testFunc := func(name string, retryAfterDuration, retryDuration time.Duration, n int, expectMinDuration time.Duration) {
```

Comment:
Would you mind updating test cases with `f-tests` approach like in rest of tests in VM?
Please see https://itnext.io/f-tests-as-a-replacement-for-table-driven-tests-in-go-8814a8b19e9e

Reply:
done.